### PR TITLE
Revert toolchain setup

### PR DESCRIPTION
**Pull request by tmattio**
The automatic installation of Platform tools caused issues for a lot of users. While we still want to provide an automated setup to our users, the current version is not flexible enough and does not support enough workflows.

A few things that we will need to do before it is ready:

- Make the toolchain setup optional
- Add a setting for the repo of the toolchain
- Do not setup two switches
- Select sandbox before the toolchain setup
- Only update when packages are outdated. Make it optional

Until we polish it, let's revert it.

Additionally, it would be good to test large changes with the community the next time we want to release them. @Khady @peter-jerry-ye @voodoos @tominated @lubegasimon, you've all reported issues with this release. Would you be willing to be part of a testing group for the next release? If you know people in your respective organizations who would be, feel free to ask them to reply to this PR, or reach out at thibaut.mattio@gmail.com

### CODE REVIEW

**Comment by voodoos** [comment]
> Additionally, it would be good to test large changes with the community the next time we want to release them. Khady peter-jerry-ye voodoos tominated lubegasimon, you've all reported issues with this release. Would you be willing to be part of a testing group for the next release?

I can do that. Note that I usually use VSCodium on Macos.

**Comment by tominated** [comment]
I'm willing to test. I'm using vscode on windows through WSL for context. Thanks for sorting this out so quickly!

**Comment by Khady** [comment]
@tmattio I’m happy to give a hand. But I’d also like to say that the current status already looks pretty good. Nothing that I have reported is currently a dealbreaker in my environment

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## Unreleased
+
+- Revert automatic installation of platform tools
+
+  The feature is not flexible enough and caused issues with multiple users. We
+  will work on making it more flexible and fit more workflow before releasing
```
Comment (mnxn) [inline CHANGELOG.md:8]: Typo
```suggestion
  will work on making it more flexible and fit more workflows before releasing
```

**Review by mnxn — APPROVED** [review]

**Pull request merged**